src/sum.js

```diff
@@ -0,0 +1,282 @@
+            "ccStockLevelStatus": "inStock",
+            "hdStockLevelStatus": "inStock"
+        },
+        "configurationInfos": [],
+        "statusSummaryMap": {},
+        "entryGroupNumbers": [
+            3
+        ],
+        "comments": [],
+        "supportedActions": [
+            "REMOVE"
+        ],
+        "onlineReturnPossible": false,
+        "departmentCode": "851",
+        "totalItemPrice": {
+            "currencyIso": "AUD",
+            "value": 3,
+            "formattedValue": "$3.00"
+        },
+        "dealsApplied": false,
+        "dealsMessage": {
+            "couldFireMessages": [],
+            "firedMessages": []
+        }
+    },
+    {
+        "entryNumber": 5,
+        "quantity": 2,
+        "basePrice": {
+            "currencyIso": "AUD",
+            "value": 12,
+            "formattedValue": "$12.00"
+        },
+        "totalPrice": {
+            "currencyIso": "AUD",
+            "value": 24,
+            "formattedValue": "$24.00"
+        },
+        "product": {
+            "code": "60942920",
+            "name": "Disney Frozen Insulated Backpack",
+            "configurable": false,
+            "brand": "Disney Frozen",
+            "inStock": false,
+            "topLevelCategory": "Kids",
+            "baseName": "Disney Frozen Insulated Backpack",
+            "baseProductCode": "P60942920",
+            "displayOnly": false,
+            "excludeForAfterpay": false,
+            "excludeForZipPayment": false,
+            "installmentPrice": {
+                "currencyIso": "AUD",
+                "value": 3,
+                "formattedValue": "$3.00"
+            },
+            "originalCategoryCode": "W97404",
+            "eligibleForAdvFreeDelivery": false,
+            "productType": "normal",
+            "imageUrl": "/medias/static_content/product/images/hero/70/91/A1147091.jpg",
+            "totalInterest": 0,
+            "maxThresholdInterest": 0,
+            "showStoreStockForProduct": false,
+            "assorted": false,
+            "productDisplayType": "AVAILABLE_FOR_SALE",
+            "hdStock": 2,
+            "ccStock": 2,
+            "inNetwork": true,
+            "ccStockLevelStatus": "inStock",
+            "hdStockLevelStatus": "inStock",
+            "colour": "Blue"
+        },
+        "configurationInfos": [],
+        "statusSummaryMap": {},
+        "entryGroupNumbers": [
+            4
+        ],
+        "comments": [],
+        "supportedActions": [
+            "REMOVE"
+        ],
+        "onlineReturnPossible": false,
+        "departmentCode": "160",
+        "totalItemPrice": {
+            "currencyIso": "AUD",
+            "value": 24,
+            "formattedValue": "$24.00"
+        },
+        "dealsApplied": false,
+        "dealsMessage": {
+            "couldFireMessages": [],
+            "firedMessages": []
+        }
+    }
+]
+
+
+function cartvalue() {
+    let tsum = 0;
+    for (let i = 0; i < t.length; i++) {
```

Comment:
use array.reduce instead for loop
reference - https://www.w3schools.com/js/js_array_iteration.asp